ivy_tests/test_ivy/test_frontends/test_tensorflow/test_signal.py

```diff
@@ -9,36 +9,29 @@
 # kaiser_window
 @handle_frontend_test(
     fn_tree="tensorflow.signal.kaiser_window",
-    dtype_and_window_length=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("integer")
-    ),
-    dtype_and_beta=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("numeric")
-    ),
-    dtype=helpers.get_dtypes("numeric"),
+    window_length=helpers.ints(min_value=1, max_value=100),
+    beta=helpers.floats(min_value=0.0, max_value=80),
     test_with_out=st.just(False),
 )
 def test_tensorflow_kaiser_window(
     *,
-    dtype_and_window_length,
-    dtype_and_beta,
-    dtype,
+    window_length,
+    beta,
     frontend,
     test_flags,
     fn_tree,
     on_device,
 ):
-    window_length_dtype, window_length = dtype_and_window_length
-    beta_dtype, beta = dtype_and_beta
+    window_length = window_length
```

Comment:
These two statements are just duplications and re-assignments, kindly remove these.